Entities/Course.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class Course : BaseEntity
+    {
+        public string CourseName { get; set; }
+        public List<Timetable> Timetables { get; set; }
```

Comment:
There is list of timetables in course? Not just one?

Reply:
OK